pkg/cloud/gcp/actuators/machine/reconciler.go

```diff
@@ -248,7 +248,10 @@ func (r *Reconciler) setMachineCloudProviderSpecifics(instance *compute.Instance
 	r.machine.Labels[machinecontroller.MachineAZLabelName] = r.providerSpec.Zone
 
 	if r.providerSpec.Preemptible {
-		r.machine.Labels[machinecontroller.MachineInterruptibleInstanceLabelName] = ""
+		if r.machine.Spec.Labels == nil {
```

Comment:
tangential to this PR I'd expect this label to be set as `r.machine.Labels` as well so it can be used for filtering. @JoelSpeed thoughts?

Reply:
I think it was put here so that it would reflect onto the nodes, we could move it to `r.machine.labels` if those will also get reflected on the node, can't remember quite how that's done, nodelink? (Or we just put it on both here)

Reply:
Yes, node link https://github.com/openshift/machine-api-operator/blob/master/pkg/controller/nodelink/nodelink_controller.go#L239